web-client/package.json

```diff
@@ -26,7 +27,8 @@
     "remark-parse": "^10.0.1",
     "remark-rehype": "^10.1.0",
     "styled-jsx-plugin-postcss": "^4.0.1",
-    "unified": "^10.1.1"
+    "unified": "^10.1.1",
```

Comment:
Also not sure what this package is for. Does this do anything? Can we remove it?

Reply:
Apparently it's used in one of the boilerplate pages. (https://github.com/noiph/hurado/blob/main/web-client/src/utils/Markdown.ts)